logics/Cargo.toml

```diff
@@ -1,21 +1,16 @@
 [package]
 name = "payable_mint_pkg"
-version = "0.2.0"
-authors = ["Stake Technologies <devops@stake.co.jp>"]
+version = "0.3.0"
+authors = ["Astar builder"]
 edition = "2021"
 
 [dependencies]
-ink_primitives = { version = "3.4.0", default-features = false }
-ink_metadata = { version = "3.4.0", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "3.4.0", default-features = false }
-ink_storage = { version = "3.4.0", default-features = false }
-ink_lang = { version = "3.4.0", default-features = false }
-ink_prelude = { version = "3.4.0", default-features = false }
+ink = { version = "~4.0.0", default-features = false}
 
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
-scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
```

Comment:
version `2` is same as having latest `2.x`, unless we want to pin the dependency to `2.3`.

Reply:
getting many compile problems. Will leave it at 2.3

Reply:
@ashutoshvarma can you approve?

Reply:
I think I've already approved.